### DISCUSSION TITLE
Video background page builder with breeze blank theme

### DISCUSSION THREAD

**Discussion by JordanMade** [breezefront/community#250] [1. Help] [answered]
Hello, 

We are running 

swissup/module-breeze 2.12.0 and
swissup/theme-frontend-breeze-blank on 2.6.0 

When I try to put the YouTube link as a background in our cms page (Page Builder), I don't have any error but it always shows the fallback image.

In the backend the video is showing correctly in the page builder but not in frontend,

Do you have any tips ?

Thanks

**Comment by JordanMade** [comment]
I can give you access to our preprod, do you have an ip I can whitelist ? and eventually tell me where I can give you login / password for our BO if you need ? 

Thanks for your help

**Reply by vovayatsyuk** [reply]
You can contact swissup support and we will provide you IP address: https://swissuplabs.com/contacts/

**Comment by vovayatsyuk** [comment, marked as answer]
The issue is caused by [ConsentManager.net](https://www.consentmanager.net/). It blocks YouTube scripts from loading on the site:

```html
<script data-cmp-src="https://www.youtube.com/iframe_api" src="" class="null cmplazyload" data-cmp-info="4" data-cmp-vendor="unknownVendor"></script>
```

**Reply by JordanMade** [reply]
Thanks a lot, it works like a charm now !
